Source/WebCore/Modules/geolocation/Geolocation.cpp

```diff
@@ -227,7 +227,7 @@ void Geolocation::resumeTimerFired()
     }
 
     if (m_errorWaitingForResume) {
-        handleError(*m_errorWaitingForResume);
+        handleError(Ref { m_errorWaitingForResume.releaseNonNull() });
```

Comment:
Something is really fishy here. If we're calling releaseNonNull(), we shouldn't have to manually construct Ref?

Reply:
This was an accidental amalgamation of a few things I tried, it should just be releaseNonNull(). I cleaned it up